frontend/src/components/bs/components/spaces-table/components/action-cell/action-cell.tsx

```diff
@@ -0,0 +1,11 @@
+import deleteIcon from 'assets/img/delete-icon.svg';
+import { IconButton } from 'components/common/common';
+
+const ActionCell = (
+  id: string,
+  onSpaceDelete: (id: string) => void,
+): JSX.Element => {
+  return <IconButton id={id} icon={deleteIcon} onAction={onSpaceDelete} />;
```

Comment:
```suggestion
  return <IconButton id={id} iconName={IconName.TRASh} onAction={onSpaceDelete} />;
```
IconName should be an enum